src/content/en/2020/javascript.md

```diff
@@ -285,23 +314,42 @@ Once again, this appears to be an area where third-party scripts are actually do
 
-Thankfully, that's exactly what we see, particularly in third-party scripts where 90% of uncompressed scripts are less than 5kb in size. On the other hand, 49% of uncompressed first-party scripts are less than 5kb and 37% of uncompressed first-party scripts are over 10kb. So while we do see a lot of small uncompressed first-party scripts, there are still quite a few that would benefit from some compression.
+{{ figure_markup(
+  caption="Percent of uncompressed third-party JavaScript requests under 5 KB.",
+  content="90.25%",
+  classes="big-number",
+  sheets_gid="347926930",
+  sql_file="compression_none_by_bytes.sql"
+) }}
+
+Thankfully, that's exactly what we see, particularly in third-party scripts where 90% of uncompressed scripts are less than 5 KB in size. On the other hand, 49% of uncompressed first-party scripts are less than 5 KB and 37% of uncompressed first-party scripts are over 10 KB. So while we do see a lot of small uncompressed first-party scripts, there are still quite a few that would benefit from some compression.
 
 ## What do we use?
+
 As we've increasingly used more JavaScript to power our sites and applications, there has also been an increasing demand for open-source libraries and frameworks to help with improving developer productivity and overall code maintainability. Sites that _don't_ wield one of these tools are definitely the minority on today's web—jQuery alone is found on nearly 85% of the mobile pages tracked by HTTP Archive.
 
 It's important that we think critically about the tools we use to build the web and what the trade-offs are, so it makes sense to look closely at what we see in use today.
 
 ### Libraries
-HTTP Archive uses Wappalyzer to detect technologies in use on a given page. Wappalazyer tracks both JavaScript Libraries (think of these as a collection of snippets or helper functions to ease development, like jQuery) and JavaScript Frameworks (these are more likely scaffolding and provide templating and structure, like React).
+
+HTTP Archive uses [Wappalyzer](./methodology#wappalyzer) to detect technologies in use on a given page. Wappalazyer tracks both JavaScript libraries (think of these as a collection of snippets or helper functions to ease development, like jQuery) and JavaScript frameworks (these are more likely scaffolding and provide templating and structure, like React).
 
 The popular libraries in use are largely unchanged from last year, with jQuery continuing to dominate usage and only one of the top 21 libraries falling out (lazy.js, replaced by DataTables). In fact, even the percentages of the top libraries has barely changed from last year.
 
-{# table? showing rank, library, percentage and last years rank #}
+{# TODO(analysts): table? showing rank, library, percentage and last years rank #}
```

Comment:
This requires pulling 2019 data so a bit more involved. I wonder if the figure below gets the job done. @tkadlec LMK.

Reply:
Sorry...what figure is that?

Reply:
https://almanac.httparchive.org/en/2020/javascript?#fig-21

(The changes in this PR were just deployed to production)

Reply:
Gotcha. I think that's fine. A comparison is kinda boring anyway with the lack of change.